examples/alphaTestExamples/helloapp.md

```diff
@@ -0,0 +1,128 @@
+[hello]: https://github.com/monopole/hello
+[kind]: https://github.com/kubernetes-sigs/kind
+
+# Demo: hello app
+
+This demo helps you to deploy an example hello app end-to-end using kustomize.
+
+Steps:
+1. Create the resources files.
+3. Spin-up kubernetes cluster on local using [kind].
+4. Deploy the app using kapply and verify the status.
+
+First define a place to work:
+
+<!-- @makeWorkplace @testE2EAgainstLatestRelease-->
+```
+DEMO_HOME=$(mktemp -d)
+```
+
+Alternatively, use
+
+> ```
+> DEMO_HOME=~/hello
+> ```
+
+## Establish the base
+
+Let's run the [hello] service.
+
+<!-- @createBase @testE2EAgainstLatestRelease-->
+```
+BASE=$DEMO_HOME/base
+mkdir -p $BASE
+OUTPUT=$DEMO_HOME/output
+mkdir -p $OUTPUT
+```
+
+Now lets add a simple config map resource to the `base`
+
+<!-- @createConfigMapYaml @testE2EAgainstLatestRelease-->
+```
+cat <<EOF >$BASE/configMap.yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: the-map
+data:
+  altGreeting: "Good Morning!"
+  enableRisky: "false"
+EOF
+```
+
+Create `service.yaml` pointing to the deployment created above
+
+<!-- @createServiceYaml @testE2EAgainstLatestRelease-->
+```
+cat <<EOF >$BASE/service.yaml
+kind: Service
+apiVersion: v1
+metadata:
+  name: the-service
+spec:
+  selector:
+    deployment: hello
+  type: LoadBalancer
+  ports:
+  - protocol: TCP
+    port: 8666
+    targetPort: 8080
+EOF
+```
+
+Create a `grouping.yaml` resource. By this, you are defining the grouping of the current directory, `base`. Kustomize uses the unique label in this file to track any future state changes made to this directory. Make sure the label key is `kustomize.config.k8s.io/inventory-id` and give any unique label value and DO NOT change it in future.
```

Comment:
Please wrap long text so the lines are under 80 chars for easy edit and review.
It makes no difference to the rendering.

The first `Kustomize` should be `kapply`, and the 2nd should be `cli-utils.sigs.k8s.io/inventory-id`